### DISCUSSION TITLE
Why alist show admin's password in plaintext?

### DISCUSSION THREAD

**Discussion by liero1982** [AlistGo/alist#1924] [General]
show password of admin in plaintext is very dangerous. 
I think most people will install this software on personal computers instead of servers. It only takes a short time to get the password.
And then, people who stold the password can log in and manage through the website on the LAN.
So please disable this high-risk feature.

我想能用alist聚合网盘功能的人多半不是把alist架设在机房服务器里的，更多的使用场景是用在个人电脑上。也就是说，心怀不轨之人可以利用很短的时间获得alist管理员的明文密码。加上alist运行方式就是web端管理，知道密码之后就可以在局域网里以管理员身份登陆，从而有充分的时间窃取文件。
所有，请禁用“alist password”这个危险命令吧。

**Comment by ihxjie** [comment]
you should take care of your computer, not alist.
